packages/jest-each/src/bind.ts

```diff
@@ -18,16 +18,20 @@ export type EachTests = Array<{
   arguments: Array<unknown>;
 }>;
 
-type TestFn = (done?: Global.DoneFn) => Promise<any> | void | undefined;
-type GlobalCallback = (testName: string, fn: TestFn, timeout?: number) => void;
+// type TestFn = (done?: Global.DoneFn) => Promise<any> | void | undefined;
+type GlobalCallback = (
+  testName: string,
+  fn: Global.ConcurrentTestFn,
+  timeout?: number,
+) => void;
 
-export default (cb: GlobalCallback, supportsDone: boolean = true) => (
-  table: Global.EachTable,
-  ...taggedTemplateData: Global.TemplateData
-) =>
+export default <A extends Global.TestCallback>(
```

Comment:
What is `A`? Can you give it a more descriptive name?

Reply:
Renamed it to EachCallback